packages/pilot/src/containers/CreateTransaction/Payment/CreditCard.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Form from 'react-vanilla-form'
+import {
+  CardContent,
+  CardSection,
+  CardTitle,
+  Checkbox,
+  Col,
+  FormDropdown,
+  FormInput,
+  Row,
+} from 'former-kit'
+
+import isCVV from '../../../validation/cvv'
+import isCreditCard from '../../../validation/creditCard'
+import isRequired from '../../../validation/required'
+import isFutureDate from '../../../validation/sameOrFutureDate'
+
+import style from './style.css'
+
+const validations = t => ({
+  cvv: [isRequired(t('required')), isCVV('cvv_invalid')],
+  expirationDate: [
+    isRequired(t('required')),
+    isFutureDate(t('date_invalid'), { format: 'MM/YY' }),
```

Comment:
```suggestion
    isFutureDate(t('date_invalid'), { format: 'MM/YY', period: 'month' }),
```

aqui precisa passar o periodo como mes senão o mes atual fica como invalido 
<img width="245" alt="screen shot 2018-12-27 at 15 32 21" src="https://user-images.githubusercontent.com/479495/50489100-f5a9f980-09ec-11e9-9367-8faa83d4e10c.png">

Reply:
boa 👏 só não vou commitar a sugestão porque quero fazer adicionar no mesmo commit